app/helpers/rubygems_helper.rb

```diff
@@ -62,35 +62,40 @@ def unsubscribe_link(rubygem)
             else
               'display:none'
             end
-    link_to t('.links.unsubscribe'), rubygem_subscription_path(rubygem),
+    link_to t('rubygems.show.links.unsubscribe'), rubygem_subscription_path(rubygem),
       class: [:toggler, 'gem__link', 't-list__item'], id: 'unsubscribe',
       method: :delete, remote: true, style: style
   end
 
   def atom_link(rubygem)
-    link_to t(".links.rss"), rubygem_versions_path(rubygem, format: 'atom'),
+    link_to t('rubygems.show.links.rss'), rubygem_versions_path(rubygem, format: 'atom'),
       class: 'gem__link t-list__item', id: :rss
   end
 
   def download_link(version)
     link_to_page :download, "/downloads/#{version.full_name}.gem"
   end
 
+  def reverse_dependencies_link(rubygem)
+    link_to_page :reverse_dependencies, rubygem_reverse_dependencies_path(rubygem)
+  end
+
   def documentation_link(version, linkset)
     return unless linkset.nil? || linkset.docs.blank?
     link_to_page :docs, version.documentation_path
   end
 
   def badge_link(rubygem)
     badge_url = "https://badge.fury.io/rb/#{rubygem.name}/install"
-    link_to t(".links.badge"), badge_url, class: "gem__link t-list__item", id: :badge
+    link_to t("rubygems.show.links.badge"), badge_url,
+      class: "gem__link t-list__item", id: :badge
```

Comment:
@spk You can change this back to one line to fix rubocop error. We can refactor `RubygemsHelper` in a different PR.